packages/wonder-blocks-modal/util/maybe-get-portal-mounted-modal-host-element.test.js

```diff
@@ -113,7 +113,7 @@ describe("maybeGetPortalMountedModalHostElement", () => {
                     // Assert
                     expect(result).toBeTruthy();
 
-                    const modalPortalElement = result && result.parentElement;
+                    const modalPortalElement = result;
```

Comment:
Huh, interesting.

Why did the test need to change? This implies to me that the method is not returning the right element as the portal element should be nested in something, even if it's just the body.

Reply:
I think it's because this new way of doing the portal removes an extra `div` that was originally there. (`maybeGetPortalMountedModalHostElement` was returning the `firstElementChild` but now doesn't do that)

Reply:
It was doing that because, at least for the tooltip, if we didn't, the tooltip appeared obscured by the modal backdrop. If that's no longer happening, then awesome.

Reply:
The dropdown portal is the last child of the backdrop.  @wishdasher I'm curious to see if this affects your keyboard navigation, hopefully it won't.

![screen shot 2018-08-15 at 3 50 01 pm](https://user-images.githubusercontent.com/1044413/44169582-e8761900-a0a2-11e8-9533-bafc9fcb9ee2.png)

Reply:
It does :( but even before this change, it was already the last child of the backdrop. There's some wonky behavior with the FocusTrap that I'm planning on looking into after the keyboard nav is merged (meaning that pull request #268 doesn't work with the modal but I am aware of it).

Reply:
Interesting... maybe we can introduce another container that lives inside the tab trap and that's where we add the modal dialog, any portals, as well as the left/right bumpers whenever we get around to adding them.

Reply:
I'm going to work on the proposal I made in my last comment.

Reply:
Awesome! I have a bit of extra context. When I was doing this, one issue I ran into was that the dropdown had to be a sibling of the dialog to be able to extend past the dialog div, but then these lines in `FocusTrap`
```
        const modalRoot = this._getModalRoot();
        if (modalRoot.contains(target)) {
            // If the newly focused node is inside the modal, we just keep track
            // of that.
            this._lastNodeFocusedInModal = target;
        } else {
```
weren't able to trap the focus events happening on the dropdown, which is why if you look at the MultiSelect in a modal on netlify, clicking anything in the dropdown focuses the close icon.

I briefly tried changing `_getModalRoot` to get its parent instead, but I think that messed up the "trap" part and tabbing was able to escape to the actual browser.

Reply:
I have a fix in https://github.com/Khan/wonder-blocks/pull/283.